src/main/java/de/tum/cit/aet/artemis/core/service/ArtemisSuccessfulLoginService.java

```diff
@@ -97,22 +99,31 @@ private boolean isEmptyOrDefaultLink(final Optional<String> link) {
      * Handles successful authentication events.
      * Sends a login notification email to users when they successfully authenticate.
      *
-     * @param username             the username of the user who has successfully logged in
+     * @param loginOrEmail         the username or email of the user who has successfully logged in
      * @param authenticationMethod the method used for authentication
      * @param clientEnvironment    the environment information of the client (optional)
      * @see AuthenticationMethod for available authentication methods
      */
-    public void sendLoginEmail(String username, AuthenticationMethod authenticationMethod, @Nullable ClientEnvironment clientEnvironment) {
+    public void sendLoginEmail(String loginOrEmail, AuthenticationMethod authenticationMethod, @Nullable ClientEnvironment clientEnvironment) {
+        String lowercaseLoginOrEmail = loginOrEmail.toLowerCase(Locale.ENGLISH);
```

Comment:
We do we need to make it lowercase? The EMail method `findOneByEmailIgnoreCase` suggests that it ignores the casing anyways and for my understanding the email regex should work with a potential uppercase version aswell?